charts/spire/charts/spire-server/templates/_helpers.tpl

```diff
@@ -105,3 +105,49 @@ Create the name of the service account to use
 {{- end }}
 {{- end }}
 
+{{- define "spire-server.config-mysql-query" }}
+{{- $lst := list }}
+{{- range . }}
+{{- range $key, $value := . }}
+{{- $eValue := toString $value }}
+{{- $entry := printf "%s=%s" (urlquery $key) (urlquery $eValue) }}
```

Comment:
If it was just the = it would be a lot easier. the = and the urlquery makes it much trickier. Not sure how to reduce it without being very similar in line count (maybe slightly less) but with more convolution. The printf join line is a bit different too.

Reply:
got it, no need to change anything. thanks for the explanation.